### ISSUE TITLE
Resisting CreepJS/Evasion Methods

### ISSUE THREAD

**Issue by JaneJeon** [abrahamjuliot/creepjs#175]
First of all, I just want to say that this project is _amazing_ - I have a scraping setup with fingerprint injection and real Chrome that I use for scraping _real_ websites (and it has been very successful in doing so), but CreepJS literally just left my setup naked and completely exposed. It's absolutely god-tier.

That said, I want to use this issue to advance "the other side" of the arms race and discuss what methods one might be able to use to evade creepJS.

For example, I am able to hide everything from creepjs except for navigator (which I thought was also injected), where creepjs detects the lie very easily. Any ideas for how one would be able to evade this?

**Comment by abrahamjuliot** [comment]
Glad you like 👍. Special thanks to the many [research works](https://github.com/abrahamjuliot/web-fingerprinting) that continue to inspire the project.

Advancing evasion methods is outside the scope of this project, but I think it's great if others want to share and discuss here or point to a better source.

**Comment by JaneJeon** [comment]
Definitely, just wanted to put some heads together and use this as a jumping off point. Thanks for understanding!

**Comment by abrahamjuliot** [comment]
I'm guiding this over to the discussion topic section. If you have any specific issues/questions on this topic, let's discuss.

As a more focused source, here's a nice repo.

https://github.com/niespodd/browser-fingerprinting
